src/export_dxf.cc

```diff
@@ -35,61 +35,84 @@
 void export_dxf(const Polygon2d &poly, std::ostream &output)
 {
 	setlocale(LC_NUMERIC, "C"); // Ensure radix is . (not ,) in output
+
 	// Some importers (e.g. Inkscape) needs a BLOCKS section to be present
-	output << "  0\n"
-				 <<	"SECTION\n"
-				 <<	"  2\n"
-				 <<	"BLOCKS\n"
-				 <<	"  0\n"
-				 << "ENDSEC\n"
-				 << "  0\n"
-				 << "SECTION\n"
-				 << "  2\n"
-				 << "ENTITIES\n";
+	output  << "  0\n" << "SECTION\n"
+			<< "  2\n" << "BLOCKS\n"
+			<< "  0\n" << "ENDSEC\n";
+
+	output  << "  0\n" << "SECTION\n"
+			<< "  2\n" << "ENTITIES\n";
+
+	// Some importers (e.g. Inkscape v1.1) needs a layer to be specified, but just once is enough
+	output  << "  8\n" << "0\n"; // layer 0
```

Comment:
This change seems to break LibreCAD import.

Reply:
Updated the code based on LibreCAD's output.
Can you send the link to the test files breaking LibreCAD in case it still breaks?

I understand the need to add extra stuff to be compatible with other softwares even though they are not necessary to be conformant. But I just wanted to keep it minimal to avoid incompatibilities with other softwares. The old version does not work with some versions of Illustrator.

Since you are open to this change, I will go test with other softwares.

Reply:
It seems DXF format need more info that what is currently generated.
I checked various dxf files/libraries and applied those for 2D.

This one is working on app I have access to.
It outputs complete dxf header for 2D.

btw, so of the comments in the code is from the original, such as what Inkscape needs, etc.